src/android/android-wizard.ts

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import * as fs from 'fs';
+// @ts-ignore - clack is ESM and TS complains about that. It works though
+import * as clack from '@clack/prompts';
+import * as path from 'path';
+import * as gradle from './gradle';
+import * as manifest from './manifest';
+import * as codetools from './code-tools';
+import {
+  abort,
+  confirmContinueEvenThoughNoGitRepo,
+  getOrAskForProjectData,
+  printWelcome,
+} from '../utils/clack-utils';
+import { WizardOptions } from '../utils/types';
+import { traceStep, withTelemetry } from '../telemetry';
+import chalk from 'chalk';
+
+export async function runAndroidWizard(options: WizardOptions): Promise<void> {
+  return withTelemetry(
+    {
+      enabled: options.telemetryEnabled,
+      integration: 'android',
+    },
+    () => runAndroidWizardWithTelemetry(options),
+  );
+}
+
+async function runAndroidWizardWithTelemetry(
+  options: WizardOptions,
+): Promise<void> {
+  printWelcome({
+    wizardName: 'Sentry Android Wizard',
+    promoCode: options.promoCode,
+  });
+
+  await confirmContinueEvenThoughNoGitRepo();
+
+  const projectDir = process.cwd();
+  const buildGradleFiles = findFilesWithExtensions(projectDir, [
+    '.gradle',
+    'gradle.kts',
+  ]);
+
+  if (!buildGradleFiles || buildGradleFiles.length === 0) {
+    clack.log.error(
+      'No Gradle project found. Please run this command from the root of your project.',
+    );
+    await abort();
+    return;
+  }
+
+  const appFile = await traceStep('Select App File', () =>
+    gradle.selectAppFile(buildGradleFiles),
+  );
+
+  // ======== STEP 1. Add Sentry Gradle Plugin to build.gradle(.kts) ============
+  clack.log.info(
+    `Adding ${chalk.bold(
+      'Sentry Gradle plugin',
+    )} to your app's build.gradle file.`,
```

Comment:
I've changed info/step messages to use `cyan`, but when it was successful, we already use `greenBright` for the entire message, so I just kept `bold` there, otherwise it looked like a rainbow :D 
<img width="475" alt="image" src="https://github.com/getsentry/sentry-wizard/assets/4999776/f4a93d5f-3821-42a8-b421-4d27e4d7778d">

Reply:
Sorry for causing confusing, I agree we shouldn't create rainbows (leave that up to the rain) 😅 

What I meant was that I'd show the filename (in this case `build.gradle`) in cyan, not the plugin. Maybe this doesn't make sense in the Android wizard case - I'll leave this up to you. 

As for using colours we previously tried to adhere to a few simple (but unfortunately unwritten) rules. We should define this properly somewhere, to make creating wizards easier. I'll chop down some notes here - maybe we can add them to a better place somewhere later on:

There are definitely exceptions to these cases but generally:
* print major success messages in green
* use `clack.log.(success|error|warn|info)` with their respective styles
* print filenames in cyan

For showing copy/paste code instructions (e.g. see sourcemaps wizard) we
* show surronding (unchanged code) in `chalk.gray`
* show code that needs changes/added code in `chalk.greenbBright`+
* Directly `console.log` the code snippet so that it's as easy as possible to copy/paste (using `clack.*` prepends the clack UI characters in front of the code snippet).

Reply:
oh, sorry, I completely misread about filenames 🤦 I will change the filename to be in cyan, that makes sense!